app/src/modernXr/java/app/gamenative/launch/LaunchReadinessInstaller.kt

```
@@ -0,0 +1,7 @@
+package app.gamenative.launch
+
+import android.content.Context
+import kotlinx.coroutines.CoroutineScope
+
+fun installLaunchReadiness(context: Context, scope: CoroutineScope) {
+}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)